attributes/default.rb

```diff
@@ -85,6 +85,8 @@
 
 # Chef Inspec Compliance profiles to be used for scan of node
 # See README.md for details
+# TODO: Make this `{}` in next major version
+# See: https://github.com/chef-cookbooks/audit/pull/328
 default['audit']['profiles'] = []
```

Comment:
I am not sure but removing examples will not resolve the main issue of hash of hashes #328  as we still having default as array.

Reply:
Correct, I was not aware Hash of Hashes was broken. We are going to leave the current docs for now, then when we get Hash of Hashes working we will make sure we include setting the parent to `{}` in all examples.

Reply:
we'll also need to update the Automate 2 examples for when this is released as they use the `.push` method as well

Reply:
@teknofire can you add any links to https://github.com/chef-cookbooks/audit/issues/359

Want to make sure we track them in all the places.

Reply:
https://automate.chef.io/docs/data-collection/#setting-up-chef-client-to-send-compliance-scan-data-through-the-chef-server-to-chef-automate

Reply:
https://github.com/chef/automate/blob/master/components/automate-chef-io/content/docs/data-collection.md